Fixed Translation Demo not working in 4.0-dev

voylin opened the pull request:
Fixes #703

Re-import of translations was needed, fixed project settings, added the remaps of audio and flags, fixed dependencies, changed the font on the corresponding nodes and added the translations in the project settings.
Everything is working as expected now

aaronfranke reviewed (commented):
When I click the Spanish button and click the button to play audio, it plays the English sound.

Additionally, we should be setting demos to Vulkan Mobile if it doesn't look any different compared to Vulkan Clustered. Especially for GUI-only demos, they will look the same in Vulkan Mobile. Go to Project -> Project Settings -> General tab -> Rendering section -> Vulkan -> Back End -> Select Forward Mobile.

voylin commented:
> When I click the Spanish button and click the button to play audio, it plays the English sound.

Sorry about that, small mistake when typing, wrote "se" instead of "es".
Fixed the sound and made the project run on Vulkan Mobile. Squashed and pushed. ^^

aaronfranke reviewed (approved):
Thanks!

> Sorry about that, small mistake when typing, wrote "se" instead of "es".

Yup, that's why I always say the code doesn't work until you test it :)
